ScnSoft-Education/LeetCodeTraining/MedianOfSortedArrays/MedianOfSortedArraysSolution.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace LeetCodeTraining.MedianOfSortedArrays
+{
+    //https://leetcode.com/problems/median-of-two-sorted-arrays/
+    public static class MedianOfSortedArraysSolution
+    {
+        public static double FindMedianSortedArrays(int[] nums1, int[] nums2)
+        {
+            int[] resultingArray = new int[nums1.Length + nums2.Length];
+
+            for (int i = 0; i < nums1.Length; i++)
+            {
+                resultingArray[i] = nums1[i];
+            }
+
+            for (int i = nums1.Length; i < resultingArray.Length; i++)
+            {
+                resultingArray[i] = nums2[i - nums1.Length];
+            }
+
+            Array.Sort(resultingArray);
```

Comment:
It is hard problem and one of the requirements is The overall run time complexity should be O(log (m+n)).
So it should be resoled ideally to follow the statements as described here https://stackoverflow.com/questions/2307283/what-does-olog-n-mean-exactly:
The most common attributes of logarithmic running-time function are that:

![image](https://github.com/daniilnaumovich/ScnSoft-Education/assets/101803780/29b0ecef-945f-410e-92c8-42976de9fe4e)

and I believe at least without using Sorting, because initial arrays already sorted, please try to implement the addition solution without sorting merged arrays and restrictions above.